canonical-web-and-design/microk8s.io
# Update content

## Done
- Update content to match the copy doc
- Add prettier config file
- Make tab heading reveal correctly
- Improve small screen layout 

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the content matches [the copy doc](https://docs.google.com/document/d/1ObWmhRc4GCIhrBi8gT7yIIs0J_dn_8ravfLtcwhr3uQ/edit#)
- Check small screen view is better
- Ensure thhe tab heading "Click for install steps" works as you would expect

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/7682
